src/transformers/models/bart/modeling_tf_bart.py

```diff
@@ -1279,7 +1299,6 @@ def __init__(self, config, load_weight_prefix=None, *inputs, **kwargs):
         self.bias_layer = BiasLayer(
             name="final_logits_bias", shape=[1, config.vocab_size], initializer="zeros", trainable=False
         )
-        self.final_logits_bias = self.bias_layer.bias  # alias to keep the same interface with PT
```

Comment:
this alias was leaving traces of the original bias when it was overwritten, causing weight saving to fail (two variables with the same name)